src/cli/util.js

```diff
@@ -613,7 +613,13 @@ function createDetailedUsage(context, optionName) {
       ? `\n\nDefault: ${createDefaultValueDisplay(optionDefaultValue)}`
       : "";
 
-  return `${header}${description}${choices}${defaults}`;
+  const pluginDefaults =
+    option.pluginDefaults && Object.keys(option.pluginDefaults).length
+      ? `\nPlugin defaults:${Object.keys(option.pluginDefaults).map(
+          key => `\n* ${key}: ${option.pluginDefaults[key]}`
```

Comment:
We should call `createDefaultValueDisplay(value)` here, though there is no non-primitive value in core options.